great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -332,7 +333,7 @@ def get_batch_data_and_markers(
             # pyspark will raise an AnalysisException error if path is incorrect
             except pyspark.sql.utils.AnalysisException:
                 raise ExecutionEngineError(
-                    f"""Unable to read in batch from the following path: {path}. Please check your configuration."""
+                    f"""Unable to read in batch from the following: {reader_path}. Please check your configuration."""
```

Comment:
@lccasagrande It still says `reader_path` instead of `path` when I am looking at it.  Thanks.

Reply:
Done